### PR TITLE
Create combined onboarding flow for NO

### PR DESCRIPTION
**Pull request by mehdielket**

### CODE REVIEW

File: angel-data/onboarding-NO.json [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+    },
+    {
+      "height": 100,
+      "id": "39267c62-d7a5-40ed-99e7-6598c8ab91ac",
+      "left": 250,
+      "name": "Additional Coverage",
+      "selected": false,
+      "story": "a115f38e-75d6-44bc-9163-41332615776d",
+      "tags": [],
+      "text": "<Message>\n  Choose Additional Coverage!\n</Message>\n\n<SelectAction>\n  <Option key=\"travel\"\n          value=\"true\"\n  >\n    [[Travel->Start]]\n  </Option>\n  <Option key=\"accident\"\n          value=\"true\"\n  >\n    [[Accident->Start]]\n  </Option>\n  <Option key=\"no-addon\"\n          value=\"true\"\n  >\n    [[None->Start]]\n  </Option>\n</SelectAction>",
```
Comment (robinandeer) [inline angel-data/onboarding-NO.json:131]: Not translated?

**Review by robinandeer — COMMENTED** [review]
Hmm looks like a few things are not translated...

File: angel-data/onboarding-NO.json [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+  "keywords": {
+    "backButton": "Back",
+    "externalInsuranceProviderConfirmTitle": "",
+    "selectActionSelectLabel": "Select",
+    "tooltipModalInformationLabel": "Information"
+  },
+  "lastUpdate": "2022-03-16T10:30:01.458Z",
+  "locales": [
+    "en_NO"
+  ],
```
[the remark this answers is not in the document]
Comment (guilhermespopolin) [reply angel-data/onboarding-NO.json]: How to configure that 🤔 ?

[the remark this answers is not in the document]
Comment (robinandeer) [reply angel-data/onboarding-NO.json]: I think you just have to manually edit the JSON file

**Review by guilhermespopolin — DISMISSED** [review]

**Review by guilhermespopolin — APPROVED** [review]

**Review by robinandeer — APPROVED** [review]
Let's try it out!

**Comment by guilhermespopolin** [comment]
> Let's try it out!

Since this flow uses chained redirect we'd need to get [this one](https://github.com/HedvigInsurance/embark/pull/825) merged first

**Pull request merged**